native/src/modules/layout/components/Header.js

```diff
@@ -66,7 +65,7 @@ export type PropsType = {|
   categoriesAvailable: boolean,
   goToLanguageChange?: () => void,
   routeCityModel?: CityModel,
-  language: string,
+  language: mixed,
```

Comment:
Why mixed?

Reply:
Flow is complaining, because 

Cannot return object literal because mixed [1] is incompatible with string [2] in property language.

flow-typed/npm/@react-navigation/stack_v5.x.x.js
[1] 428│   declare export type ScreenParams = { +[key: string]: mixed, ... };

Any better idea to fix this, than chaning to mixed?

Reply:
I'd rather use a typeof check (or a flow fix me) than having a mixed type for something that is always a string. What do you think?

Reply:
You are completly right, a typeof check is much nicer. I added it.